components/Ticket/index.js

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+
+export default function Ticket({ ticketValue, id, onDeleteTicket }) {
+  return (
+    <StyledTicket>
+      <div>
+        <StyledValue>{ticketValue}€</StyledValue>
+      </div>
```

Comment:
Why do you need this `div` at this point?

```suggestion
        <StyledValue>{ticketValue}€</StyledValue>
```

Reply:
i had it for a reason and then forgot to delete it, thanks Stefan